nginx/nginx.conf

```diff
@@ -83,5 +83,5 @@ http {
         text/css
         text/plain;
 
-    include /etc/nginx/conf.d/*.conf;
+    include conf.d/*.conf;
```

Comment:
how does this account for `conf.d/django.conf.d/locations/static.conf` ?

Reply:
ah, nm, found that line.

Reply:
It doesn't it just includes `conf.d/django.conf` and that includes the things in `conf.d/django.conf.d`.